app/helpers/eu_exit_finder.rb

```diff
@@ -0,0 +1,5 @@
+module EUExitFinderHelper
```

Comment:
I'd suggest a few changes around this helper:

- Rename `eu_exit_finder.rb` to match the module name (i.e. `eu_exit_finder_helper.rb`, then you won't need the `require eu_exit_finder` lines in the presenters, since Rails autoloads them
- Lowercase `EUExitFinderHelper` to `EuExitFinderHelper`, otherwise the file would have to be named `e_u_exit_finder_helper.rb` as Rails splits on the changing case in the module name
- Replace the method name from `EUExitFinderHelper.eu_exit_finder?` to `eu_exit_finder?` and update the calls in the presenters, then the `eu_exit_finder?` method in the finder presenter could also be removed

Reply:
That last bulletpoint is tricky, as all existing calls to `eu_exit_finder?` rely on the presenter holding the state of the content ID, but my new `eu_exit_finder?` helper takes a content ID as an argument.

Quite happy to keep the existing `eu_exit_finder?` method as a façade for my `eu_exit_finder?` method and change less code.